requre/online_replacing.py

```diff
@@ -81,3 +84,38 @@ def _internal(*args, **kwargs):
         return _internal
 
     return decorator_cover
+
+
+def store(what: str, in_module: str):
+    """
+    Decorator which can be used to store calls of the function and
+    and replay responses on the next run.
+    """
+
+    def _store_inner(func):
+        return replace(
+            what=what, in_module=in_module, decorate=Simple.decorator_all_keys
```

Comment:
This is not so clean, or better to say, this `Simple` class is intended and could be used to store and restore just simple objects, eg, string, numbers, bools, and dict and lists of these basic types. So it is not generic enough, but at least output is readable, you can use `ObjectStorage` from `objects` what use pickle module to store and restore. It is little bit cryptic, ale least stored output, but it is more generic and allows to store and restore any serializable object.

This is also reason that I'm little bit against these generic simplification.
second problem is that using `decorator_all_keys`, will stored all parameters of method as keys, what is little bit insecure if don't know it. maybe better to use `decorator_plain` what will store no param as keys.

Reply:
- Thanks, users are not expected to read the output -- we can use the `pickle` implementation.

> This is also reason that I'm little bit against these generic simplification.
second problem is that using decorator_all_keys, will stored all parameters of method as keys, what is little bit insecure if don't know it. maybe better to use decorator_plain what will store no param as keys.

I would leave this to the user and make this optional:
- For `requests` we can have some options to allow `header` filtering.
- For general decorators, we can have some value filtering. (`filter_key="some_regex"`, `filter_value="other_regex"`)

I would start with some simplification and add new functionality only if requested/needed. We don't need to make it easily usable now and some more advanced features can be added later (and ideally optional).

Reply:
sure, could be. just ensure, that it is clear for users, that it is not generic enough, because it can be used just in case seriazable objects, so anythong what does some file streaming or process running, will ends up with strange issues. E.g. this is reasong why `git` module needs own wrapper, becuase it process git subprocesses.

for `requests` I meant that I generally fix it in `RequestResponseHandling` class to filter this out and then there we can use `decorator(item_list=["url"])` and then URL will be stored, what is probably the most important, filtering other values like params, data, headers is harder in general.  or better to say, to create some syntax what will be easy to users and powerfull enough..

Reply:
> sure, could be. just ensure, that it is clear for users, that it is not generic enough, because it can be used just in case seriazable objects, so anythong what does some file streaming or process running, will ends up with strange issues. E.g. this is reasong why `git` module needs own wrapper, becuase it process git subprocesses.

OK, I'll play with that and maybe create multiple versions, like `store_seriazable`, `store_and_match_arguments`. Or maybe introduce some enum for `match_mode`.

> for requests I meant that I generally fix it in RequestResponseHandling class to filter this out and then there we can use decorator(item_list=["url"]) and then URL will be stored, what is probably the most important, filtering other values like params, data, headers is harder in general. or better to say, to create some syntax what will be easy to users and powerfull enough..

It's hard to say what is a good default. What about `match_data=False` and `match_headers=False` arguments?